assets/css/main.css

```diff
@@ -19,6 +21,11 @@
     --secondary: var(--purple);
 }
 
-* {
-    font-family: 'Roboto', sans-serif;
+html, body {
+    height: 100%;
+    width: 100%;
+    margin: 0;
+    padding: 0;
+    color: var(--black);
+    font-family: 'Inter';
```

Comment:
font by default is still should be Roboto

Reply:
Fixed